assets/ui.css

```
@@ -9,6 +9,10 @@
   background: transparent;
 }
 
+.respec-modal .close-button:focus {
```

```suggestion
.respec-modal .close-button:focus:not(:focus-visible) {
```

Because we do want to show outline when navigating through keyboard buttons.

---

@saschanaz this doesn't seem to solve the problem of close button being focused initially

---

Ah yes, [no browser support yet](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#Browser_compatibility)

As it's not breaking anything, I would just wait for the browser support rather than just permanently disabling outline.